pkg/config/envvar/envvar_test.go

```diff
@@ -117,6 +119,8 @@ instances:
      q6: 'foo'
      q7: 'foo#bar'
      q8: ['foo#bar', 'baz' ]  
+     m1: |-
+       multiline # string
```

Comment:
I think the expected output should remove the " # string" comment, like:
```
m1: |-
       multiline 
```

Reply:
My bad, indeed the comment should not be stripped in yaml block scalars: https://jsonformatter.org/yaml-to-json/d2a266